tests/core/test_playback.py

```diff
@@ -491,6 +491,35 @@ def test_next_emits_events(self, listener_mock):
             ],
             listener_mock.send.mock_calls)
 
+    def test_next_emits_events_when_consume_mode_is_enabled(self,
+                                                            listener_mock):
```

Comment:
I'd break the line  after `(` here and indent `self` one level more than the following `tl_tracks ...` line. This is however still flake8 compliant, so I won't stand my ground.